### DISCUSSION TITLE
Gradient as a border

### DISCUSSION THREAD

**Discussion by imyanice** [ImageMagick/ImageMagick#5456] [Help] [answered]
Hi, I want to create an image with a gradient border like the following image:
![c5ac357fb06568334a](https://user-images.githubusercontent.com/76485792/185372924-23e2bb70-a32f-4386-86fe-d08dac6d8991.png) (the picture is just black im talking about the border). I made some researches but I found nothing, so if you can help me...
Have a beautiful day !

**Comment by imyanice** [comment]
Version: ImageMagick 7.1.0-45 Q16-HDRI aarch64 20319 https://imagemagick.org, macOS

**Comment by fmw42** [comment, marked as answer]
One way is to create a background color gradient image and then draw a black rectangle over the background gradient.

```
magick \
\( -size 320x420 gradient:red-blue -rotate -90 \) \
-fill black -draw "rectangle 10,10 410,310" -alpha off \
result.png

```

![result](https://user-images.githubusercontent.com/13458218/185426934-49692c50-b40e-472f-9a24-a25fad9c8d53.png)

**Reply by imyanice** [reply]
Thanks a lot !
